tests/contrib/botocore/test.py

```diff
@@ -693,12 +696,90 @@ def test_lambda_invoke_bad_context_client(self):
         spans = self.get_spans()
         assert spans
         span = spans[0]
-        self.assertEqual(len(spans), 1)
-        self.assertEqual(span.get_tag("aws.region"), "us-west-2")
-        self.assertEqual(span.get_tag("aws.operation"), "Invoke")
+        assert len(spans) == 1
+        assert span.get_tag("aws.region") == "us-west-2"
+        assert span.get_tag("aws.operation") == "Invoke"
         assert_is_measured(span)
         lamb.delete_function(FunctionName="black-sabbath")
 
+    @mock_events
+    def test_eventbridge_single_entry_trace_injection(self):
+        bridge = self.session.create_client("events", region_name="us-west-2")
+        bridge.create_event_bus(Name="a-test-bus")
+
+        Pin(service=self.TEST_SERVICE, tracer=self.tracer).onto(bridge)
+
+        entries = [
+            {
+                "Source": "some-event-source",
+                "DetailType": "some-event-detail-type",
+                "Detail": json.dumps({"foo": "bar"}),
+                "EventBusName": "a-test-bus",
+            }
+        ]
+        bridge.put_events(Entries=entries)
+
+        spans = self.get_spans()
+        assert spans
+        assert len(spans) == 1
+        span = spans[0]
+
+        str_entries = span.get_tag("params.Entries")
+        entries = ast.literal_eval(str_entries)
```

Comment:
Can you leave a comment on this, using the `ast` here is not obvious. An example of the what the tag looks like would be good.

(maybe refactor the span -> entries into a shared function?)

Reply:
Added some comments to explain